app/serializers/developer.js

```diff
@@ -0,0 +1,27 @@
+import RESTSerializer from 'ember-data/serializers/rest'
```

Comment:
It did work without a serializer, but I got warnings when I made calls to my backend. I consulted @sandersky, who helped me implement a serializer. We suspected that the calls I was making didn't specify the primary key of **username**, so a warning was thrown.